### ISSUE TITLE
err  Repo angular not found!

### ISSUE THREAD

**Issue by lookfirst** [jspm/jspm-cli#632]
https://travis-ci.org/lookfirst/systemjs-seed/jobs/56042263

Is there an easier way to figure out which project has the dependency on 'angular'? It would be nice if the output included that.

I suspect it is the [ngReact project](https://github.com/davidchang/ngReact/blob/master/package.json).

This used to work fine prior to 0.15. Another interesting thing is that this only broke once I put it up on CI. I had to locally clear my `.jspm/` folder to get it to reproduce. I guess that is what `-f` is for.

**Comment by guybedford** [comment]
It's tricky to note which dependency is doing the install as a single install operation can have many sources, but I will certainly note that for future debugging.

For now I managed to make the error message slightly more useful and provide the message - `err  Repo github:angular not found!` instead.

Doing a good old debugging by elimination showed it to be https://github.com/jspm/registry/blob/b5d09cdce043a2c3a64d9b2334df8aacf831c99c/package-overrides/github/angular/bower-angular-mocks%401.3.0.json.

I've been working on simplifying the rules behind how packages resolve, and this one seems to have been lost in translation.

Basically I removed having overrides dependencies defaulting to also overriding the registry to the jspm registry when not otherwise set, which is very confusing. So we either need to set `registry: 'jspm'` or use an explicit name (which is what I'm trying to encourage as I don't think the jspm registry should be used for these types of installs, since we don't promise resolutions of the naming service won't change over time).

Fixed in https://github.com/jspm/registry/commit/2b70f86604781d171320e3aa10bbb4811d7be6b5.
